spec/Section 7 -- Response.md

```diff
@@ -10,7 +10,12 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A response to a GraphQL request must be a map.
+A GraphQL request returns either a _response_ or a _response stream_.
+
+### Response
+
+:: A GraphQL request will return a _response_ when the GraphQL operation is a
```

Comment:
Maybe more of a personal thing but I tend to avoid future tense in spec/docs. I like that it "sounds" more universal and is generally easier to read. But I just checked the spec and there's a fair amount of future tense there so take this with a pinch of salt.

```suggestion
:: A GraphQL request returns a _response_ when the GraphQL operation is a
```

Reply:
Agreed, updated